src/Database/Schema/PostgresSchemaDialect.php

```diff
@@ -765,7 +765,10 @@ public function columnDefinitionSql(array $column): string
         if (
             isset($column['default']) &&
             in_array($column['type'], $datetimeTypes) &&
-            strtolower($column['default']) === 'current_timestamp'
+            (
+                strtolower($column['default']) === 'current_timestamp' ||
+                str_contains(strtolower($column['default']), 'now')
```

Comment:
```suggestion
                strtolower($column['default']) == 'now()'
```

Should we be more precise in what the acceptable values are?

Reply:
well, currently we specially handle these kind of cases since they should not be output as strings.

But If i understand the `Literal` class correctly, it should be used to put exactly what you store into it in the place you reference it.

So doing `'default' => Literal::from('now()'),` should result in just `now()` being output, not `'now()'`.

Doing manual if checks here for these kind of special cases feels wrong since it should be the users choice what to put there.

Reply:
Yes, for `Literal` expressions we shouldn't do any additional checks, and can use the value inside the `Literal`.